.github/CODEOWNERS

```diff
@@ -2,7 +2,7 @@
 
 * @baentsch
 /oqsprov/oqs_hyb_kem.c @bhess
-/oqs-template/generate.yml @baentsch @bhess @feventura
+/oqs-template/generate.yml @baentsch @bhess @feventura @alexrow
```

Comment:
we'll need an update to the [governance yaml](https://github.com/open-quantum-safe/tsc/blob/main/config.yaml) to support this user